android/src/main/java/org/mozilla/mozstumbler/client/mapview/tiles/CoverageOverlay.java

```diff
@@ -16,18 +16,27 @@
  * This class provides the Mozilla Coverage overlay
  */
 public class CoverageOverlay extends AbstractMapOverlay {
+    public static final String MLS_MAP_TILE_COVERAGE_NAME = "Mozilla Location Service Coverage Map";
     // Use a lower zoom than the LowResMapOverlay, the coverage can be very low detail and still look ok
-    private static final int LOW_ZOOM_LEVEL = 10;
+    public static final int LOW_ZOOM_LEVEL = 10;
+
+    public CoverageOverlay(final Context aContext, final String coverageUrl, MapView mapView) {
+        super(aContext);
+        setup(1, AbstractMapOverlay.LARGE_SCREEN_MIN_ZOOM, coverageUrl, mapView);
```

Comment:
Yes, the relevant range is only from `LOWEST_UNLIMITED_ZOOM` to `LOW_ZOOM_LEVEL - 1` (3 to 9). But as I said above, restricting the range at this part of the code seemed unnecessary to me. But I can change that if it improves code readability.